docs/docs/10-overview.md

```diff
@@ -21,7 +21,7 @@ Kargo's goal is to provide an intuitive and flexible layer "above" existing GitO
 Watch the *Multi-Stage Deployment Pipelines the GitOps Way* talk by Jesse Suen & Kent Rancourt of Akuity at GitOpsCon EU 2024.
 
 <center>
-<iframe width="560" height="315" src="https://www.youtube.com/embed/tl8w72tehZ0?si=fdSx63nsCQHR1eP0" title="YouTube video player" frameBorder="0" allow="accelerometer; autoplay; clipboard-write; encrypted-media; gyroscope; picture-in-picture;" allowFullScreen></iframe>
+<iframe width="1280" height="720" src="https://www.youtube.com/embed/0B_JODxyK0w" title="Kargo - Multi-Stage Deployment Pipelines using GitOps - Jesse Suen / Kent Rancourt" frameborder="0" allow="accelerometer; autoplay; clipboard-write; encrypted-media; gyroscope; picture-in-picture; web-share" referrerpolicy="strict-origin-when-cross-origin" allowfullscreen></iframe>
```

Comment:
You've resized this to be too big even for the max column width.

Even the old width was too big on some smaller viewports.

Wondering if @rbreeze could give some guidance here to make this iframe constrained by the width of the containing element.